x-pack/plugins/reporting/export_types/csv/server/lib/generate_csv.js

```diff
@@ -14,22 +14,19 @@ export function createGenerateCsv(logger) {
   const hitIterator = createHitIterator(logger);
 
   return async function generateCsv({
```

Comment:
Was the intent here to alphabetize?

Reply:
Trying to declare variables closer to where the are used.

There will be some conflicts from https://github.com/elastic/kibana/pull/29977 in this part

Reply:
267cd9179d